.syncpackrc

```diff
@@ -266,6 +264,25 @@
 			],
 			"pinVersion": "wp-6.6"
 		},
+		{
+			"dependencies": [
+				"@wordpress/babel-preset-default",
+				"@wordpress/babel-plugin-import-jsx-pragma"
+			],
+			"packages": [
+				"**"
+			],
+			"pinVersion": "wp-6.6"
+		},
+		{
+			"dependencies": [
+				"@babel/**"
+			],
+			"packages": [
+				"**"
+			],
+			"pinVersion": "7.x.x"
```

Comment:
I usually pin the first two parts of the version, like` 7.26.x`, to avoid frequent modifications of the lock file and related merge conflicts. Not pushing it here, as I don't have 100% context here and it's rather something to consider.

Reply:
There isn’t really any context to this, I was trying various things out blindly to try to resolve some react-18 breakage and decided not to waste the work even though it didn’t fix it 😅

I’ll switch it to 7.26.x